keyboards/mode/eighty/m80s/mcuconf.h

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
If these ChibiOS conf files are identical for both versions, they can go at the top level too.